src/controllers/stallController.ts

```diff
@@ -138,6 +138,14 @@ async function indexStall(req: Request, res: Response, next: NextFunction) {
     const page = +req.query.page!;
     const offset = (page - 1) * limit;
 
+    if (!limit || !page) {
```

Comment:
I think we should set a logical default limit and page and run it through the pagination methods. This will make sure that the API response is consistent on the stalls route.